src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,126 @@
-import { useState } from 'react';
+import React, { useEffect, useState } from 'react';
+import { Movie } from '../../types/Movie';
 import { TextField } from '../TextField';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
```

Comment:
it's better to combine all movie's field to one use state, as it will make a code shorter and cleaner 
```suggestion
 const initMovie = {
  title: '',
  ...
 }
 const [movie, setMovie] = useState(initMovie);
```